README.md

```diff
@@ -13,19 +13,27 @@ Install packages
 Import modules
 
 ```python
-from azure.identity import XCredential
+from azure.identity import UsernamePasswordCredential, DeviceCodeCredential
 from msgraphcore import GraphSession, AuthorizationHandler, AuthMiddlewareOptions, TokenCredentialAuthProvider
 ```
 
 Configure Credential Object
 
 ```python
-browser_credential = XCredential(<tenant_id>, <client_id>)
+# Added UsernamePassword for demo purposes only, please don't use this in production.
+# ugly_credential = UsernamePasswordCredential('set-clientId', 'set-username', 'set-password')
+
+device_credential = DeviceCodeCredential(
+    'set-clientId')
+
+# There are many other options for getting an access token. See the following for more information.
+# https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/azure/identity/_credentials/__init__.py
```

Comment:
Do you think it will be better to link to the azure.identity documenation instead? Source files or their location might change. 
https://pypi.org/project/azure-identity/

Reply:
Agreed. Changed.